packages/utils/telemetry-utils/src/mockLogger.ts

```diff
@@ -71,6 +71,19 @@ ${JSON.stringify(actualEvents)}`);
 expected:
 ${JSON.stringify(expectedEvents)}
 
+actual:
+${JSON.stringify(actualEvents)}`);
+            }
+    }
+
+    /** Asserts that matchAnyEvent is false for the given events, and prints the actual/expected output if not */
+    assertMatchNone(disallowedEvents: Omit<ITelemetryBaseEvent, "category">[], message?: string) {
+        const actualEvents = this.events;
+        if (this.matchAnyEvent(disallowedEvents)) {
+            throw new Error(`${message}
+disallowed events:
+${JSON.stringify(disallowedEvents)}
```

Comment:
Didn't know this was here, this is cool

Reply:
Yeah there's more I want it to do, just adding bit by bit :)